### PR TITLE
`<bs4.BeautifulSoup>` object ai choan cho `<str>`, fix #365

### LINKED ISSUE #365: Applying sekchi.0008_換掉海口腔變調 有錯誤
```
Running migrations:
  Applying sekchi.0005_sekchi_對齊狀態... OK
  Applying sekchi.0006_alter_sekchi_對齊狀態... OK
  Applying sekchi.0007_sekchi_口語調狀態... OK
  Applying sekchi.0008_換掉海口腔變調...Traceback (most recent call last):
  File "manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/__init__.py", line 446, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.8/site-packages/django/core/management/__init__.py", line 440, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/base.py", line 414, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/base.py", line 460, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/base.py", line 98, in wrapped
    res = handle_func(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/commands/migrate.py", line 290, in handle
    post_migrate_state = executor.migrate(
  File "/usr/local/lib/python3.8/site-packages/django/db/migrations/executor.py", line 131, in migrate
    state = self._migrate_all_forwards(
  File "/usr/local/lib/python3.8/site-packages/django/db/migrations/executor.py", line 163, in _migrate_all_forwards
    state = self.apply_migration(
  File "/usr/local/lib/python3.8/site-packages/django/db/migrations/executor.py", line 248, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/usr/local/lib/python3.8/site-packages/django/db/migrations/migration.py", line 131, in apply
    operation.database_forwards(
  File "/usr/local/lib/python3.8/site-packages/django/db/migrations/operations/special.py", line 193, in database_forwards
    self.code(from_state.apps, schema_editor)
  File "/app/sekchi/migrations/0008_換掉海口腔變調.py", line 15, in haikhau_piantiau
    obj.save()
  File "/usr/local/lib/python3.8/site-packages/django/db/models/base.py", line 806, in save
    self.save_base(
  File "/usr/local/lib/python3.8/site-packages/django/db/models/base.py", line 857, in save_base
    updated = self._save_table(
  File "/usr/local/lib/python3.8/site-packages/django/db/models/base.py", line 970, in _save_table
    updated = self._do_update(
  File "/usr/local/lib/python3.8/site-packages/django/db/models/base.py", line 1034, in _do_update
    return filtered._update(values) > 0
  File "/usr/local/lib/python3.8/site-packages/django/db/models/query.py", line 881, in _update
    query.add_update_fields(values)
  File "/usr/local/lib/python3.8/site-packages/django/db/models/sql/subqueries.py", line 113, in add_update_fields
    val = val.resolve_expression(self, allow_joins=False, for_save=True)
TypeError: 'NoneType' object is not callable
```

### PR DESCRIPTION
**Pull request by niauah**

### CODE REVIEW

File: sekchi/migrations/0008_換掉海口腔變調.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def haikhau_piantiau(apps, schema_editor):
         # if we directly import it, it'll be the wrong version
         Sekchi = apps.get_model("sekchi", "Sekchi")
         for obj in Sekchi.objects.all():
-            obj.羅馬字含口語調 = piann_haikhau_piantiau(obj.羅馬字含口語調)
+            obj.羅馬字含口語調 = str(piann_haikhau_piantiau(obj.羅馬字含口語調))
```
Comment (sih4sing5hong5) [inline sekchi/migrations/0008_換掉海口腔變調.py:14]: 按呢應該是tī `piann_haikhau_piantiau`就ài轉`str()`?別人用較袂有仝款問題

另外，伊敢bô對應`str()`ê 函式？

**Review by sih4sing5hong5 — COMMENTED** [review]
:+1:

**Comment by sih4sing5hong5** [comment]
#365

File: sekchi/migrations/0008_換掉海口腔變調.py (hunk 1 above)
Comment (niauah) [reply sekchi/migrations/0008_換掉海口腔變調.py:14]: 我是考慮kāng module內底有 `clean_html` ，傳--ê mā是 `<bs4.BeautifulSoup>`。因為後壁別的function koh需要用`<bs4.BeautifulSoup>` 做代誌。

https://github.com/i3thuan5/SuiSiann-KauTui/blob/d765e1d3c3d5f926c73d8eb9e7bfd0f978410b63/sekchi/models.py#L57-L61

**Comment by niauah** [comment]
https://www.crummy.com/software/BeautifulSoup/bs4/doc/#non-pretty-printing

> If you just want a string, with no fancy formatting, you can call str() on a BeautifulSoup object, or on a Tag within it:

File: sekchi/migrations/0008_換掉海口腔變調.py (hunk 1 above)
Comment (niauah) [reply sekchi/migrations/0008_換掉海口腔變調.py]: 93ecb8c 改好--ah

**Review by sih4sing5hong5 — APPROVED** [review]
按呢調整API攏是`str`後，確實有較好讀 :+1:

**Pull request merged**